.vscode/tasks.json

```diff
@@ -2,16 +2,143 @@
+                "/property:GenerateFullPaths=true"
+            ],
+            "problemMatcher": "$msCompile"
+        },
+        {
+            "label": "Build MonoGame Framework (WindowsDX)",
+            "command": "dotnet",
+            "type": "process",
+            "dependsOn": ["Build MonoGame Framework Content Pipeline"],
+            "args": [
+                "build",
+                "${workspaceFolder}/MonoGame.Framework/MonoGame.Framework.WindowsDX.csproj",
+                "/property:GenerateFullPaths=true"
+            ],
+            "problemMatcher": "$msCompile"
+        },
+        {
+            "label": "Build MonoGame Framework (WindowsUniversal)",
+            "command": "dotnet",
+            "type": "process",
+            "dependsOn": ["Build MonoGame Framework Content Pipeline"],
+            "args": [
+                "build",
+                "${workspaceFolder}/MonoGame.Framework/MonoGame.Framework.WindowsUniversal.csproj",
+                "/property:GenerateFullPaths=true"
+            ],
+            "problemMatcher": "$msCompile"
+        },
+        {
+            "label": "Build MonoGame Framework Content Pipeline",
+            "command": "dotnet",
+            "type": "process",
+            "args": [
+                "build",
+                "${workspaceFolder}/MonoGame.Framework.Content.Pipeline/MonoGame.Framework.Content.Pipeline.csproj",
+                "/property:GenerateFullPaths=true"
+            ],
+            "problemMatcher": "$msCompile"
+        },
+        {
+            "label": "Build mgcb-editor",
+            "command": "dotnet",
+            "type": "process",
+            "dependsOn": ["Build MonoGame Framework Content Pipeline"],
+            "osx": {                
+                "args": [
+                    "build",
+                    "${workspaceFolder}/Tools/MonoGame.Content.Builder.Editor/MonoGame.Content.Builder.Editor.Mac.csproj",
+                    "/property:GenerateFullPaths=true",
+                    "/consoleloggerparameters:NoSummary"
+                ]
+            },
+            "linux": {
+                "args": [
+                    "build",
+                    "${workspaceFolder}/Tools/MonoGame.Content.Builder.Editor/MonoGame.Content.Builder.Editor.Linux.csproj",
+                    "/property:GenerateFullPaths=true",
+                    "/consoleloggerparameters:NoSummary"
+                ]
+            }, 
+            "windows": {
+                "args": [
+                    "build",
+                    "${workspaceFolder}/Tools/MonoGame.Content.Builder.Editor/MonoGame.Content.Builder.Editor.Windows.csproj",
+                    "/property:GenerateFullPaths=true",
+                    "/consoleloggerparameters:NoSummary"
+                ]
+            },
+            "problemMatcher": "$msCompile"
+        },
+        {
+            "label": "Run mgcb-editor",
```

Comment:
There is already a launch configuration for mgcb-editor above, also `dotnet run` is invalid for the mgcb-editors mac project, you need to use `open -a <pathtogeneratedappfile>`

Reply:
Thank you for letting me know about the `dotnet run` on Mac.